src/query/settings/src/settings_getter_setter.rs

```diff
@@ -600,6 +600,14 @@ impl Settings {
         )
     }
 
+    pub fn get_enable_refresh_inverted_index_after_write(&self) -> Result<bool> {
+        Ok(self.try_get_u64("enable_refresh_inverted_index_after_write")? != 0)
```

Comment:
I think it must be true.

Otherwise, the index query will not work and throw errors.

So do we need this setting ?

Reply:
ok, we can remove this setting and add it in the further if needed.